torch/onnx/_constants.py

```diff
@@ -8,6 +8,8 @@
 # ONNX_DEFAULT_OPSET generated by tools/onnx/update_default_opset_version.py
 ONNX_DEFAULT_OPSET = 14
 ONNX_CONSTANT_FOLDING_MIN_OPSET = 9
+# Pinned version from ATenlib
+FX_ONNX_OPSET = 17
```

Comment:
We can use 18 if PyTorch has bumped onnx

Reply:
I recall there were some reduce related regression for 18. Is that resolved? cc @titaiwangms

Reply:
I think so

Reply:
https://github.com/pytorch/pytorch/pull/96567